pylabrobot/liquid_handling/liquid_handler.py

```diff
@@ -1878,6 +1880,19 @@ async def move_plate(
       to_location = to
     elif isinstance(to, MFXModule):
       to_location = to.get_absolute_location() + to.child_resource_location
+    elif isinstance(to, PlateAdapter):
+      # Calculate location adjustment of Plate based on PlateAdapter geometry
+      x_locations = sorted(OrderedDict.fromkeys([well_n.location.x
+        for well_n in plate.children]))
+      y_locations = sorted(OrderedDict.fromkeys([well_n.location.y
+        for well_n in plate.children]))
+      plate_dx, plate_dy = x_locations[0], y_locations[0]
+      well_size_x = plate.children[0].get_size_x()
+      well_size_y = plate.children[0].get_size_y()
+      plate_x_adjustment = to.dx - plate_dx + to.adapter_hole_size_x/2 - well_size_x/2
+      plate_y_adjustment = to.dy - plate_dy + to.adapter_hole_size_x/2 - well_size_y/2
+      adjusted_plate_anchor = Coordinate(plate_x_adjustment, plate_y_adjustment, to.dz)
+      to_location = to.get_absolute_location() + adjusted_plate_anchor
```

Comment:
is there a way to make this a private function on `PlateAdapter` (eg `_compute_child_location`) to minimize code duplication between PlateAdapter and LiquidHandler?

Reply:
Yes, I think that is a great idea!

I will implement this tonight

Reply:
Implemented and tested in the lab